package.json

```diff
@@ -20,6 +20,9 @@
     "build": "rollup --configPlugin @rollup/plugin-typescript -c rollup.config.ts && cp ./src/reference/v8.json ./dist/latest.json",
     "generate-style-spec": "node --no-warnings --loader ts-node/esm build/generate-style-spec.ts",
     "generate-typings": "node --no-warnings --loader ts-node/esm build/generate-typings.ts",
+    "docs": "node --no-warnings --loader ts-node/esm build/generate-docs.ts",
+    "start-docs": "docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material",
```

Comment:
Maybe we can have it rebuild the docs automatically when `v8.json` changes.

Reply:
Maybe, I'm not sure how, I simply run `npm start-docs` in one terminal and `npm run docs` after I change the v8.json file.
Feel free to investigate...

Reply:
I found out how. :) Will push something.